ocs_ci/helpers/dr_helpers.py

```diff
@@ -549,3 +561,127 @@ def get_all_drpolicy():
+    fence_params = f'{{"spec":{{"clusterFence":"{constants.ACTION_FENCE}"}}}}'
+    drcluster_obj = ocp.OCP(resource_name=drcluster_name, kind=constants.DRCLUSTER)
+    assert drcluster_obj.patch(
+        params=fence_params, format_type="merge"
+    ), f"Failed to patch {constants.DRCLUSTER}: {drcluster_name}"
+    logger.info(f"Successfully fenced {constants.DRCLUSTER}: {drcluster_name}")
+    config.switch_ctx(restore_index)
+
+
+def enable_unfence(drcluster_name):
+    """
+    The OpenShift cluster to be Unfenced is the one where applications
+     are not currently running and the cluster that was Fenced earlier.
+
+    Args:
+        drcluster_name (str): Name of the DRcluster which needs to be fenced
+    """
+
+    logger.info(
+        f"Edit the DRCluster resource for {drcluster_name} cluster on the Hub cluster"
+    )
+    restore_index = config.cur_index
+    config.switch_acm_ctx()
+    unfence_params = f'{{"spec":{{"clusterFence":"{constants.AcTION_UNFENCE}"}}}}'
+    drcluster_obj = ocp.OCP(resource_name=drcluster_name, kind=constants.DRCLUSTER)
+    assert drcluster_obj.patch(
+        params=unfence_params, format_type="merge"
+    ), f"Failed to patch {constants.DRCLUSTER}: {drcluster_name}"
+    logger.info(f"Successfully unfenced {constants.DRCLUSTER}: {drcluster_name}")
+    config.switch_ctx(restore_index)
+
+
+def get_fence_state(drcluster_name):
+    """
+    Returns the clusterfence state of given drcluster
+
+    Args:
+        drcluster_name (str): Name of the DRcluster
+
+    Returns:
+        state (str): If drcluster are fenced: Fenced or Unfenced,
+                     else None if not defined
+
+    """
+    restore_index = config.cur_index
+    config.switch_acm_ctx()
+    drcluster_obj = ocp.OCP(resource_name=drcluster_name, kind=constants.DRCLUSTER)
+    state = drcluster_obj.get().get("spec").get("clusterFence")
+    config.switch_ctx(restore_index)
+    return state
+
+
+def validate_data_integrity(namespace, path="/mnt/test/hashfile", timeout=600):
+    """
+    Verifies the md5sum values of files are OK
+
+    Args:
+        namespace (str): Namespace where the workload running
+        path (str): Path of the hashfile saved of each files
+        timeout (int): Time taken in seconds to run command inside pod
+
+    Raises: If there is a mismatch in md5sum value or None
+
+    """
+    all_pods = get_all_pods(namespace=namespace)
+    for pod_obj in all_pods:
+        logger.info("Verify the md5sum values are OK")
+        cmd = f"md5sum -c {path}"
+        try:
+            pod_obj.exec_cmd_on_pod(command=cmd, out_yaml_format=False, timeout=timeout)
+            logger.info(f"Pod {pod_obj.name}: All files checksums value matches")
+        except CommandFailed as ex:
+            if "computed checksums did NOT match" in str(ex):
+                logger.error(
+                    f"Pod {pod_obj.name}: One or more files or datas are modified"
+                )
+            raise ex
```

Comment:
Maybe it would be better placed to DR workload module.

Reply:
done, moved